Controller/Component/WizardComponent.php

```diff
@@ -157,6 +157,13 @@ class WizardComponent extends Component {
 	public $nestedViews = false;
 
 /**
+ * Holds the root of the session key for data storage.
+ *
+ * @var string
+ */
+	public $sessionRootKey = 'Wizard';
```

Comment:
I don't remember my cake code too well, how do users customize this? Should this perhaps be an argument or an option in `initialize()`?